root/server/routes/orders.js

```diff
@@ -1,26 +1,33 @@
-import {Fawn} from 'fawn';
-import {Order, validate} from '../models/order';
-import {Customer} from '../models/customer';
-import {Product} from '../models/product';
-import mongoose from 'mongoose';
-import {Router} from 'express';
-const router = Router();
+const {Order, validate} = require('../models/order');
+const {Customer} = require('../models/customer');
+const {Product} = require('../models/product');
+const mongoose = require('mongoose');
+<<<<<<< HEAD
+const pick = require('lodash');
+const express = require('express');
+const router = express.Router();
+=======
+const Fawn = require('fawn');
+const pick = require('lodash');
+const express = require('express');
+const router = express.Router();
 
 Fawn.init(mongoose);
```

Comment:
a to nie mialobyc usuniete?

Reply:
miało, nie wiem czemu tu jest

Reply:
zaraz zerknę

Reply:
mój błąd, zapomniałam zapisać pliku przed commitem, dlatego takie głupoty